input/data/2011/2011A%26A...533A.103H/tev-000064.yaml

```diff
@@ -1,4 +1,25 @@
 source_id: 64
 reference_id: 2011A&A...533A.103H
 
-# TODO: add data
+data:
+  livetime: 10
+  significance: 10
+
+pos:
+  ra: {val: 13h56m0s, err: 2m, err_sys: }
+  dec: {val: -64d30m0s, err: 2m, err_sys: }
+
+morph:
+  type: gauss
+
+  sigma: {val: 0.08, err: }
+  sigma2: {val: 0.08, err: }
+
+spec:
+  type: pl
+
+  norm: {val: 2.7, err: 0.9, err_sys: 0.2}
```

Comment:
`err_sys: 1.0` printed on Figure 2, and `err_sys: 0.4` given in the text.
Don't know where you got `err_sys: 0.2` from.

Suggest to take the value from the text, and note the inconsistency in a one-line comment above the `norm: ` line (not as a TODO: doesn't matter so much).